infra/base-images/base-runner/coverage

```diff
@@ -111,8 +111,21 @@ function run_fuzz_target {
   fi
 }
 
+GO_TARGET="false"
 # Run each fuzz target, generate raw coverage dumps.
 for fuzz_target in $FUZZ_TARGETS; do
+  # Test if fuzz target is a golang one.
+  grep "go test -v -coverprofile" $fuzz_target > /dev/null 2>&1 && export GO_TARGET="true"
```

Comment:
Maybe we can have a better way to know if we are dealing with a golang project...

Reply:
Yes, we now have a `language` attribute in every project.yaml file, see https://github.com/google/oss-fuzz/blob/475f86dbc4f245c3ce9b329176e2603654246d6e/infra/helper.py#L294 and how it's used